### ISSUE TITLE
[BITV] 9.2.4.7/2.2 - "Online status" dialog - The focus on the first (selected) element is not visible since it is not differentiated from the visual indication of selected state. Sighted keyboard users may have trouble selecting status since the affordance of the status controls will make them think the controls are buttons while it is in fact a radio group (which is logically correct). (9)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37011]
- [ ] Fix
- [ ] Backport

![06779e8f4f902f8124114eb681bc0567](https://user-images.githubusercontent.com/1315170/222540731-83449141-952e-42c5-b1a6-4104a611785a.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v2-n2

</details>

- [ ] 9.2.4.7/3.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v3-n3
- [ ] 9.2.4.7/4.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v4-n4
- [ ] 9.2.4.7/5.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v5-n3
- [ ] 9.2.4.7/6.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v6-n4
- [ ] 9.2.4.7/7.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v7-n3
- [ ] 9.2.4.7/8.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v8-n4
- [ ] 9.2.4.7/9.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v9-n3
- [ ] 9.2.4.7/10.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v10-n3

**Comment by JuliaKirschenheuter** [comment]
seems to be a duplicate of https://github.com/nextcloud/server/issues/36981

**Comment by michaelnissenbaum** [comment]
Hi @JuliaKirschenheuter.From my perspective, it's okay now. 

Unfortunately, I've noticed another issue. The elements in the "Status Message" list receive a poorly noticeable keyboard focus. This needs to be changed.
![CleanShot 2023-08-16 at 20 04 46@2x](https://github.com/nextcloud/server/assets/48948959/7de92466-c521-4550-a5d4-f32a824f49e5)

**Comment by JuliaKirschenheuter** [comment]
Thank you @michaelnissenbaum!

**Comment by JuliaKirschenheuter** [comment]
This issue will be closed and issue with  "Status Message" elements will be opened.
